ui/application-window.ui

```diff
@@ -50,6 +95,126 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
+                <child internal-child="accessible">
+                  <object class="AtkObject" id="stop_button-atkobject">
+                    <property name="AtkObject::accessible-name" translatable="yes">Stop recording</property>
+                  </object>
+                </child>
+                <style>
+                  <class name="destructive-action"/>
+                </style>
+              </object>
+              <packing>
+                <property name="expand">False</property>
+                <property name="fill">True</property>
+                <property name="position">1</property>
+                <property name="non_homogeneous">True</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkButton" id="record_button">
+                <property name="label" translatable="yes">_Record</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">True</property>
+                <property name="image">icon_record_button</property>
+                <property name="use_underline">True</property>
+                <property name="always_show_image">True</property>
+                <signal name="clicked" handler="on_record_button_clicked" swapped="no"/>
+                <child internal-child="accessible">
+                  <object class="AtkObject" id="record_button-atkobject">
+                    <property name="AtkObject::accessible-name" translatable="yes">Start recording</property>
+                  </object>
+                </child>
+                <style>
+                  <class name="suggested-action"/>
+                </style>
+              </object>
+              <packing>
+                <property name="expand">False</property>
+                <property name="fill">True</property>
+                <property name="position">2</property>
+                <property name="non_homogeneous">True</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkMenuButton" id="pop_format_menu">
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">True</property>
+                <property name="halign">start</property>
+                <property name="popover">pop_format</property>
+                <child>
+                  <placeholder/>
+                </child>
+              </object>
+              <packing>
+                <property name="expand">True</property>
+                <property name="fill">True</property>
+                <property name="position">3</property>
+                <property name="non_homogeneous">True</property>
+              </packing>
+            </child>
+          </object>
+        </child>
+        <child>
+          <object class="GtkBox">
```

Comment:
Instead of a GtkBox here, you can add the GtkMenuButton directly as a child of the GtkHeaderBar.

Reply:
I missed that, will try to fix after merge

Reply:
Fixed in https://github.com/phw/peek/commit/0f4a29dca333bdbd848c5722fe4e4cb7716255fb

Thanks again